### PR TITLE
airport challenge - completed jack m.

### PR DESCRIPTION
**Pull request by jackmcc08**
Completed Airport Challenge

### CODE REVIEW

File: airport_notes.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
+TDD - the creation of classes and modules to satisfy the user stories - DONE
```
Comment (cspoppuppy) [inline airport_notes.md:1]: well documented

**Review by cspoppuppy — COMMENTED** [review]
Completed in good standard, considered edge cases, full test coverage. Well documented and neat.
